config/api.yml

```diff
@@ -807,6 +807,33 @@
       :get:
       - :name: read
         :identifier: flavor_show
+  :floating_ips:
+    :description: Floating IPs
+    :identifier: floating_ip
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
Was this needed as part of this PR? I suspect that you want to add future support for other collections to have floating ips - if that's the case I would add this line only when you do that.